.docker/Dockerfile.nginx-drupal

```diff
@@ -1,10 +1,18 @@
+##
+# `nginx-drupal` image
+#
+# This image is derived from php-fpm, which lacks tools to build Drupal.
+# The files are therefore copied from the `cli` container (Dockerfile.cli) which is built first.
+#
+
 ARG CLI_IMAGE
 FROM ${CLI_IMAGE} as cli
 
 FROM govcms8lagoon/nginx-drupal
```

Comment:
I'm wondering whether having PaaS have a hard dependency on SaaS here is wise - maybe we do as we did for cli and go `FROM amazeeio/nginx-drupal:latest` - but comment that they should refer to our govcms8lagoon dockerfile for a best-practice setup (we add a couple of extra nginx configs and sanitise)?

Reply:
I agree thoroughly.

Reply:
@tobybellwood let's defer this because it would mean duplicating a number nginx configuration files into the paas scaffold.